### PR TITLE
feat: Add new rhcos and rhcosprerelease artifacts

### PR DESCRIPTION
**Pull request by 0xFelix**
**What this PR does / why we need it**:

Add rhcos 4.13 and 4.14 and rhcosprerelease 4.14 and 4.15.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

Merge after #75 

**Release note**:

```release-note
Added rhcos 4.13 and 4.14 and rhcosprerelease 4.14 and 4.15.
```

### CODE REVIEW

**Review by jcanocan — COMMENTED** [review]
Nice job! I just have one suggestion.

File: artifacts/centos/centos.go [hunk 1]
```diff
@@ -15,16 +16,17 @@ import (
...
-	Version string
-	Variant string
-	getter  http.Getter
-	Arch    string
+	Version          string
+	Variant          string
+	getter           http.Getter
+	Arch             string
+	AdditionalLabels map[string]string
 }
```
[the remark this answers is not in the document]
Comment (0xFelix) [reply artifacts/centos/centos.go:30]: I like this idea. There's more to clean up though. What do you think about a dedicated clean up PR?

[the remark this answers is not in the document]
Comment (jcanocan) [reply artifacts/centos/centos.go:30]: I agree. Let's do it in a follow up.

**Comment by jcanocan** [comment]
Looks like `pull-containerdisks-test` is not happy. Can you please take a look?

**Comment by jcanocan** [comment]
> Looks like `pull-containerdisks-test` is not happy. Can you please take a look?



> Looks like `pull-containerdisks-test` is not happy. Can you please take a look?

Answered on: https://github.com/kubevirt/containerdisks/pull/75#issuecomment-1910493550

**Comment by dominikholler** [comment]
@davidvossel How does the hosted control plane get the images? Is the rhcos in this repo still relevant?

**Comment by 0xFelix** [comment]
Closed in favor of #78

**Pull request closed without merging**